sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -65,44 +130,56 @@ export function getEnvVars(): { [key in EnvVarKeys]: any } {
+      forBrowser
+    ),
+    [EnvVarKeys.TOPIC_FILTER_NAME]: getEnvVarValue(EnvVarKeys.TOPIC_FILTER_NAME, forBrowser),
+    [EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME]: getEnvVarValue(
+      EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME,
+      forBrowser
+    ),
+    [EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME]: getEnvVarValue(
+      EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME,
+      forBrowser
     ),
-    [EnvVarKeys.QUEUE_NAME]: getEnvVarValue(EnvVarKeys.QUEUE_NAME) || "partitioned-queue",
-    [EnvVarKeys.QUEUE_NAME_NO_PARTITION]:
-      getEnvVarValue(EnvVarKeys.QUEUE_NAME_NO_PARTITION) || "unpartitioned-queue",
-    [EnvVarKeys.QUEUE_NAME_SESSION]:
-      getEnvVarValue(EnvVarKeys.QUEUE_NAME_SESSION) || "partitioned-queue-sessions",
-    [EnvVarKeys.QUEUE_NAME_NO_PARTITION_SESSION]:
-      getEnvVarValue(EnvVarKeys.QUEUE_NAME_NO_PARTITION_SESSION) || "unpartitioned-queue-sessions",
-    [EnvVarKeys.TOPIC_NAME]: getEnvVarValue(EnvVarKeys.TOPIC_NAME) || "partitioned-topic",
-    [EnvVarKeys.TOPIC_NAME_NO_PARTITION]:
-      getEnvVarValue(EnvVarKeys.TOPIC_NAME_NO_PARTITION) || "unpartitioned-topic",
-    [EnvVarKeys.TOPIC_NAME_SESSION]:
-      getEnvVarValue(EnvVarKeys.TOPIC_NAME_SESSION) || "partitioned-topic-sessions",
-    [EnvVarKeys.TOPIC_NAME_NO_PARTITION_SESSION]:
-      getEnvVarValue(EnvVarKeys.TOPIC_NAME_NO_PARTITION_SESSION) || "unpartitioned-topic-sessions",
-    [EnvVarKeys.SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME) || "partitioned-topic-subscription",
-    [EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION) ||
-      "unpartitioned-topic-subscription",
-    [EnvVarKeys.SUBSCRIPTION_NAME_SESSION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_SESSION) ||
-      "partitioned-topic-sessions-subscription",
-    [EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION]:
-      getEnvVarValue(EnvVarKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION) ||
-      "unpartitioned-topic-sessions-subscription",
-    [EnvVarKeys.TOPIC_FILTER_NAME]: getEnvVarValue(EnvVarKeys.TOPIC_FILTER_NAME) || "topic-filter",
-    [EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_SUBSCRIPTION_NAME) || "topic-filter-subscription",
-    [EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME]:
-      getEnvVarValue(EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME) ||
-      "topic-filter-default-subscription",
     [EnvVarKeys.AAD_CLIENT_ID]: getEnvVarValue(EnvVarKeys.AAD_CLIENT_ID),
     [EnvVarKeys.AAD_CLIENT_SECRET]: getEnvVarValue(EnvVarKeys.AAD_CLIENT_SECRET),
     [EnvVarKeys.AAD_TENANT_ID]: getEnvVarValue(EnvVarKeys.AAD_TENANT_ID),
     [EnvVarKeys.RESOURCE_GROUP]: getEnvVarValue(EnvVarKeys.RESOURCE_GROUP),
     [EnvVarKeys.AZURE_SUBSCRIPTION_ID]: getEnvVarValue(EnvVarKeys.AZURE_SUBSCRIPTION_ID),
     [EnvVarKeys.CLEAN_NAMESPACE]: getEnvVarValue(EnvVarKeys.CLEAN_NAMESPACE) || false
   };
-}
+}
```

Comment:
I believe eslint will have an issue with files whose last line is not just the line ending

Reply:
Updated